app/workers/gamedev_worker.rb

```diff
@@ -0,0 +1,25 @@
+class GamedevWorker
+  include Sidekiq::Worker
+
+  def perform
+    response = RubyStackoverflow.questions({
+      order: "asc",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.